docs/source/troubleshooting.md

```diff
@@ -2,10 +2,98 @@
 
 This document is under active development.
 
-## Networking
+When troubleshooting, you may see unexpected behaviors or receive an error
+message. These two lists provide links to identifying the cause of the
+problem and how to resolve it.
 
-If JupyterHub proxy fails to start:
+## Behavior problems
+- [JupyterHub proxy fails to start](troubleshooting.md#jupyterhub-proxy-fails-to-start)
+
+## Errors
+- [500 error after spawning a single-user server](troubleshooting.md#500-error-after-spawning-my-single-user-server)
```

Comment:
For in-document links, I think you can use just the hash `#500...`, and it should work within the page (both on GitHub and RTD).